eos-chown-avahi-helper-to-writers

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
```

Comment:
As mentioned, I'm not sure a boot helper unit is actually the best way to achieve this. But if you do, please add the commands directly as `ExecStart` directives in the unit, instead of adding another binary in `$PATH`

Reply:
@cosimoc Will `tmpfiles.d` entries change the permissions on an existing path? I suppose I could try that.

Reply:
It looks like `tmpfiles.d` is only meant to be used with `/var`, `/tmp` and `/run`? https://www.freedesktop.org/software/systemd/man/tmpfiles.d.html

Reply:
@smspillaz I'm pretty sure it would also work in `/etc` -- we have already examples of that in `/usr/lib/tmpfiles.d`.

Reply:
And yes, it should also change permissions on an existing path with the right flags.